config/mailinglist/rules/jdk.json

```diff
@@ -12,17 +12,21 @@
         ],
         "client": [
             "make/\\w+(Demos|X11)",
-            "make/autoconf/lib-(font|freetype|x11)",
+            "make/autoconf/lib-(alsa|cups|font|freetype|x11)",
             "make/data/(font|x11)",
             "make/jdk/src/classes/build/tools/(generatenimbus/|x11wrappergen)",
```

Comment:
Why nimbus has a slash at the end? it can be removed?

Reply:
As said in the other comment, the slash enforces a full match on a directory instead of a partial. I don't know why this particular dir has a slash when so many others do not. Feel free to rewrite as you like.